hs_tracking/migrations/0006_auto_20190407_1436.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.18 on 2019-04-07 14:36
+from __future__ import unicode_literals
+
```

Comment:
@sblack-usu @phuongdm may want to do a quick spot check on the migrations

Reply:
@alvacouch please address this.

Reply:
@engineSound @cuahsimarko Is there anything to do here? What do I need to address?

Reply:
Mark just wanted us to double check migrations.